packages/terra-site/package.json

```diff
@@ -14,7 +14,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "scripts": {
-    "start": "webpack-dev-server --progress --port ${PORT:-8080}",
```

Comment:
The `$PORT` setting's just used for Heroku deployment. I just roll it back after switching to express server.

If it's helpful, I'm fine with just keeping it.